azure-pipelines.yml

```diff
@@ -5,4 +5,4 @@
 jobs:
   - template: ./.azure-pipelines/azure-pipelines-linux.yml
   - template: ./.azure-pipelines/azure-pipelines-osx.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
\ No newline at end of file
+  #- template: ./.azure-pipelines/azure-pipelines-win.yml
```

Comment:
Let me fix that with the conda-forge.yaml but before I'll try to investigate if we can fix Azure.

Reply:
OK. I guess that flang is the reason that only AppVeyor works. You don't need to comment that part b/c AppVeyor is the Windows package provider, but it is OK to merge as-is.